flags.go

```diff
@@ -134,9 +134,10 @@ func newApp() (app *cli.App) {
 			/////////////////////////
 
 			cli.StringFlag{
-				Name:  "endpoint",
-				Value: "https://storage.googleapis.com:443",
-				Usage: "The endpoint to connect to.",
+				Name: "custom-endpoint",
+				Usage: "The endpoint to connect to - endpoint should be equivalent to the base endpoint " +
```

Comment:
The end point to connect to for fetching data. The endpoint should be equivalent to the base endpoint of GCS JSON API.